src/pages/Profile.js

```diff
@@ -76,14 +76,22 @@ export default class Profile extends React.Component {
         }
       )
       .then(
-        () => this.props.portalStore.fetchProfileCharacters(this.state.id).then(
+        () => this.props.portalStore.fetchProfileCharacters(this.state.id,0).then(
           (characters) => {
             this.setState({
               characters: characters,
             })
           }
         )
       )
+      .then(
```

Comment:
Actually both calls can be done at the same time using https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all

This would shorten await time, but is not required.